docs/development/postman/collection.json

```diff
@@ -810,16 +817,16 @@
 	],
 	"variable": [
 		{
-			"key": "CONSUMER_MANAGEMENT_URL",
```

Comment:
I would keep the variable name as it was `CONSUMER_MANAGEMENT_UR` as the path also points to `/management`